src/autoloads/EnemyUtil.gd

```diff
@@ -0,0 +1,3 @@
+extends Node
+
+var aggro_array : Array[CharacterBody2D]
```

Comment:
why not add a pair of methods to add or remove a certain element from the aggro array? that way, it's easy for an enemy to register itself `EnemyUtil.begin_aggro(self)`.